main.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"gopkg.in/go-playground/webhooks.v5/github"
+	"net/http"
+)
+
+const (
+	path = "/payload"
+)
+
+func main() {
+	hook, _ := github.New(github.Options.Secret("<GITHUBSECRET>"))
+
+	http.HandleFunc(path, func(w http.ResponseWriter, r *http.Request) {
+		payload, err := hook.Parse(r, github.PushEvent)
+		if err != nil {
+			if err == github.ErrEventNotFound {
+				// ok event wasn't one of the ones asked to be parsed
+				println("Error receiving github payload")
+				fmt.Println(err)
+
+			}
+		}
+
+		fmt.Printf("Received Payload:\n")
+		push := payload.(github.PushPayload)
+		fmt.Printf("PUSH PAYLOAD:\n %+v", push)
+
+	})
+
+	fmt.Printf("Starting Server...\n")
```

Comment:
Is this the best way to print a line in go?

Reply:
best is subjective `println`, `fmt.println` and `fmt.Printf` all technically work for this case. I think using `fmt` is useful because it prints to stdout while `println` prints to stderr

Reply:
I think “fmt.println” or “println” is more readable—bit of a nit though